modules/serverless-rn-1-20-0.adoc

```diff
@@ -19,6 +19,12 @@
 * The `kn func` CLI plug-in now uses `func` 0.20.
 
 * The Kafka broker is now available as a Technology Preview.
++
+[IMPORTANT]
+====
+The Kafka broker Technology Preview is not supported on FIPS.
```

Comment:
for translation purposes, we should call it a Technology Preview feature and not just a Technology Preview. Maybe reword this to:

The Kafka Broker, a Technology Preview feature, is currently not supported on FIPS.
or
The Kafka Broker, which is currently in Technology Preview, is not supported on FIPS.